yt_dlp/networking/_websockets.py

```diff
@@ -36,6 +41,11 @@
 if websockets_version < (12, 0):
     raise ImportError('Only websockets>=12.0 is supported')
 
+urllib3_supported = False
+urllib3_version = tuple(int_or_none(x, default=0) for x in urllib3.__version__.split('.')) if urllib3 else None
+if urllib3_version and urllib3_version >= (1, 26, 17):
+    urllib3_supported = True
+
```

Comment:
this check should be moved to dependencies.py as we discussed on discord

Reply:
will create another PR for this - out of scope for this one